src/taskapi/task.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+
+package taskapi;
+option go_package = "github.com/pachyderm/pachyderm/v2/src/taskapi";
+
+import "gogoproto/gogo.proto";
+
+enum State {
+  STATE_UNKNOWN = 0;
```

Comment:
This can just be `UNKNOWN`, the state is implicit.

Reply:
whoops